lib/edvs.js

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const brHttpsAgent = require('bedrock-https-agent');
+const {EdvClient} = require('edv-client');
+const keyResolver = require('./keyResolver');
+const kms = require('./kms');
+
+// load config defaults
+require('./config');
+const KMS_MODULE = 'ssm-v1';
+
+// module API
+const api = {};
+module.exports = api;
+
+/**
+ * Creates an EdvClient
+ *
+ *
```

Comment:
```suggestion
 * Creates a new EDV and returns an EdvClient for it.
 *
```

Reply:
Isn't this creating an EDV and returning a client for it?

Reply:
Updated https://github.com/digitalbazaar/bedrock-profile/pull/1/commits/50e0cc990e82e9a731e959ddf862e47591ee70dd#diff-a6009addeb8d1ecd1255ccaaae42d627R17